pyserini/search/_base.py

```diff
@@ -129,6 +129,14 @@
     'dpr-wq-test': JTopics.DPR_WQ_TEST,
     'dpr-squad-test': JTopics.DPR_SQUAD_TEST,
     'dpr-curated-test': JTopics.DPR_CURATED_TEST,
+    'gar-t5-nq-test-answers': JTopics.GART5_NQ_TEST_ANSWERS,
+    'gar-t5-nq-test-titles': JTopics.GART5_NQ_TEST_TITLES,
+    'gar-t5-nq-test-sentences': JTopics.GART5_NQ_TEST_SENTENCES,
+    'gar-t5-nq-test-all': JTopics.GART5_NQ_TEST_ALL,
+    'gar-t5-dpr-trivia-test-answers': JTopics.GART5_DPR_TRIVIA_TEST_ANSWERS,
+    'gar-t5-dpr-trivia-test-titles': JTopics.GART5_DPR_TRIVIA_TEST_TITLES,
+    'gar-t5-dpr-trivia-test-sentences': JTopics.GART5_DPR_TRIVIA_TEST_SENTENCES,
+    'gar-t5-dpr-trivia-test-all': JTopics.GART5_DPR_TRIVIA_TEST_ALL,
```

Comment:
Actually, what about `gar-t5-answers` followed by the `nq-test` and so on, this is motivated by preserving the original naming schemes.